### PR TITLE
Fixed issue with wrong shipping tax amount and percent values during …

### PR DESCRIPTION
**Pull request by chaikovskyia**
…express checkout

### CODE REVIEW

**Review by frodeaa — APPROVED** [review]

File: Model/ShippingCallback.php [hunk 1]
```diff
@@ -200,8 +200,8 @@ public function getOptions()
             $shippingOption = $this->shippingOptionFactory->create();
 
             $shippingOption->setAmount($shippingMethod->getPriceExclTax() * 100)
-                ->setVat(($shippingMethod->getPriceInclTax() - $shippingMethod->getPriceExclTax()) * 100)
-                ->setVatAmount(0)
+                ->setVat(0)
```
[the remark this answers is not in the document]
Comment (chaikovskyia) [reply Model/ShippingCallback.php:203]: @svnv , the vat (percentage) is calculated later in the code (between lines 217-219). The issue was that vat and vatAmount were mixed up.

[the remark this answers is not in the document]
Comment (svnv) [reply Model/ShippingCallback.php:203]: ah, i see. But it still looks like the `amount` set in `setAmount()` in line 202 is the amount without vat, shouldn't that be amount with the vat included instead?

[the remark this answers is not in the document]
Comment (chaikovskyia) [reply Model/ShippingCallback.php:203]: @svnv , I could be wrong, but I think Dintero sums up the item amount and vat amount thus receiving item price.

[the remark this answers is not in the document]
Comment (chaikovskyia) [reply Model/ShippingCallback.php:203]: @svnv ,  you were right. It has been fixed.

**Review by svnv — APPROVED** [review]

**Pull request merged**